md_api/src/main/java/com/wipro/ats/bdre/md/api/GetProcessDependency.java

```diff
@@ -49,19 +49,15 @@ public GetProcessDependency() {
      * command line notations.
      * @return This method returns list of Upstream and Downstream processes of a particular process.
      */
-    @Autowired
-    private ProcessDependencyDAO processDependencyDAO;
 
     public List<ProcessDependencyInfo> execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)